ArmaReforgerServerTool/Forms/Main.cs

```diff
@@ -75,6 +79,20 @@ public Main()
             Mod.GetScenariosForMod("591AF5BDA9F7CE8B");
         }
 
+        /// <summary>
+        /// Retrieves the application's version number.
+        /// </summary>
+        /// <returns>Formatted version string.</returns>
+        private string GetAppVersion()
```

Comment:
This could also be used to populate the About Box if put in the `FileIOManager` singleton. Please look into doing that :)